back/lib/tasks/single_use/20240531_substitute_cl_gv.rake

```diff
@@ -0,0 +1,88 @@
+namespace :gv_transition do
+  desc 'Replace Citizenlab by Go Vocal in all static pages.'
+  task :substitute_tenants, [] => [:environment] do
+    reporter = ScriptReporter.new
+    Tenant.safe_switch_each do |tenant|
+      StaticPage.all.each do |page|
+        {
+          page.title_multiloc => 'title',
+          page.top_info_section_multiloc => 'top_info_section',
+          page.bottom_info_section_multiloc => 'bottom_info_section'
+        }.each do |multiloc, attrstr|
+          multiloc.each do |locale, old_v|
+            new_v = rake_20240531_substitute_gv(old_v)
+            if new_v != old_v
+              multiloc[locale] = new_v
+              reporter.add_change(
+                old_v,
+                new_v,
+                context: { tenant: tenant.host, page: page.slug, attribute: attrstr, locale: locale }
+              )
+            end
+          end
+        end
+        if !page.save
+          reporter.add_error(
+            page.errors.details,
+            context: { tenant: tenant.host, page: page.slug }
+          )
+        end
+      end
+    end
+    reporter.report!('substitute_tenants_report.json', verbose: true)
+  end
+
+  desc 'Prepare translation files for the transition to Go Vocal'
+  task :substitute_translation_files, [] => [:environment] do
+    translation_file_patterns = [
+      'config/locales/*.yml',
+      'engines/free/email_campaigns/config/locales/*.yml',
```

Comment:
I know it's not as important, but are there any in fixtures that need replacing?

Reply:
There are some occurrences of "Citizenlab" in the fixtures, but didn't consider it necessary to change here (it could be done "asynchronously"). I'm thinking of going through all occurrences of "Citizenlab" one last time at the end of the whole transition. Currently, we still leave quite some occurrences of "Citizenlab" throughout the code. For example, it was agreed to keep the "moderator@citizenlab.co" default moderator for now.